### PR TITLE
Docker: Kill 14 layers in pwntools base images

### PR DESCRIPTION
**Pull request by kpcyrd**
We noticed that pwntools/pwntools has an excessive amount of layers, this change squashes 14 of them into a single layer.

### CODE REVIEW

**Comment by zachriggle** [comment]
AFAICT, this require re-running all commands every time anything changes, and re-downloading the entire image any time there's a new Pwntools release (vs just the last layer)

**Comment by zachriggle** [comment]
Ah, I see these are the base images, not `dev` / `beta` / `stable`

File: extra/docker/kali/Dockerfile [hunk 1]
```diff
@@ -6,21 +6,19 @@
...
-RUN apt-get install -y python-dev
-RUN pip install --upgrade pip
-RUN python -m pip install --upgrade pwntools
-RUN PWNLIB_NOTERM=1 pwn update
-
-RUN apt-get install -y sudo
-RUN useradd -m pwntools
-RUN passwd --delete --unlock pwntools
-RUN echo "pwntools ALL=(ALL:ALL) NOPASSWD: ALL" > /etc/sudoers.d/pwntools
+RUN apt-get update && apt-get install -y build-essential \
```
Comment (zachriggle) [inline extra/docker/kali/Dockerfile]: Can you wrap each of these files on the `&&` rather than making this a long line?  Other than that LGTM

**Review by zachriggle — CHANGES_REQUESTED** [review]

File: extra/docker/kali/Dockerfile (hunk 1 above)
Comment (kpcyrd) [reply extra/docker/kali/Dockerfile]: Done :)

**Review by zachriggle — APPROVED** [review]

**Pull request merged**